src/components/SearchMovie/SearchMovie.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { MoviesList } from '../MoviesList';
+import moviesFromServer from '../../api/movies.json';
+
+export class SearchMovie extends React.Component {
+  state = {
+    query: '',
+  }
+
+  queryChange = (event) => {
+    this.setState({ query: event.target.value });
```

Comment:
```suggestion
    this.setState({ query: event.target.value.toLowerCase() });
```
Лучше тут приводить к нижнему регистру